seed/views/v3/properties.py

```diff
@@ -198,6 +199,14 @@ def _get_filtered_results(self, request, profile_id):
         except EmptyPage:
             property_views = paginator.page(paginator.num_pages)
             page = paginator.num_pages
+        except DataError as e:
+            return JsonResponse(
+                {
+                    'status': 'error',
+                    'message': f'Error filtering - your data might not match the column settings data type: {str(e)}'
```

Comment:
Looking for feedback on this -- not sure what a good error message would be. Is this semi-clear?

Reply:
Overall I think it's clear. If I had to critique:
- Include the column name. "Error filtering <column_name>...". 
- Could be useful to provide a link to the column settings page in the error message

Reply:
When a correctly typed extra_data column has empty fields you will get errors.  Should (could?) we automatically handle empty and null fields by failing all filter criteria?
<img width="303" alt="Screen Shot 2022-01-06 at 2 11 43 PM" src="https://user-images.githubusercontent.com/564231/148452728-4e33d481-a0d2-42ce-babc-2caad7f0a8bc.png">

Reply:
> Include the column name.
Wish I could, but I don't think I have that information in the error object, I need to investigate this...

> Could be useful to provide a link to the column settings page in the error message
Not a bad idea, I'll see if I can add a link